django_project/certification/views/attendee.py

```diff
@@ -70,3 +74,138 @@ def get_form_kwargs(self):
             'certifying_organisation': self.certifying_organisation
         })
         return kwargs
+
+
+class CsvUploadView(FormMessagesMixin, LoginRequiredMixin,
+                    FormView):
+    """
+    Allow upload of attendees
```

Comment:
Why two lines? Make it a single line please

Reply:
Okay @timlinux